src/plugins/intel_cpu/tests/unit/shape_inference_test/identity_shape_inference_test.cpp

```diff
@@ -0,0 +1,69 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include "identity_shape_inference.hpp"
+#include "utils.hpp"
+
+using namespace ov;
```

Comment:
Can be removed?
use core full namespace for types outside `ov::intel_cpu` namespace